examples/tf-azurerm-computegroup/variables.tf

```diff
@@ -0,0 +1,59 @@
+variable "resource_group_name" {
+    description = "The name of the resource group in which the resources will be created"
+    default = "vmssrg"
+}
+
+variable "location" { 
+    description = "The location where the resources will be created"
+    default = "West US"
```

Comment:
I'd argue this shouldn't be defaulted, so that the user has to specify it (for instance, this sample wouldn't work in Germany/China or Azure Stack in the future with this location)